tests/mgbench/benchmark.py

```diff
@@ -145,9 +105,355 @@ def filter_benchmarks(generators, patterns):
     help="directory path where temporary data should " "be stored",
 )
 parser.add_argument("--no-properties-on-edges", action="store_true", help="disable properties on edges")
+
 parser.add_argument("--bolt-port", default=7687, help="memgraph bolt port")
+
+parser.add_argument(
+    "--with-authorization",
+    action="store_true",
+    default=False,
+    help="Run each query with authorization",
+)
+
+parser.add_argument(
+    "--warmup-run",
+    action="store_true",
+    default=False,
+    help="Run warmup before benchmarks",
+)
+
+parser.add_argument(
+    "--mixed-workload",
+    nargs="*",
+    type=int,
+    default=[],
+    help="""Define combination that defines the mixed workload.
+    Mixed workload can be run as a single configuration for all groups of queries,
+    Pass the positional arguments as values of what percentage of
+    write/read/update/analytical queries you want to have in your workload.
+    Running --mixed-workload 1000 20 70 10 0 will execute 1000 queries, 20% write,
+    70% read, 10% update and 0% analytical.
+
+    Mixed workload can also be run on each query under some defined load.
+    By passing one more positional argument, you are defining what percentage of that query
+    will be in mixed workload, and this is executed for each query. The rest of the queries will be
+    selected from the appropriate groups
+    Running --mixed-workload 1000 30 0 0 0 70, will execute each query 700 times or 70%,
+    with the presence of 300 write queries from write type or 30%""",
```

Comment:
If I understand correctly the first sentence, how the mixed-workload is interpreted depends on the other arguments (= single configuration VS defined load). If I am correct, would it make sense to provide a command/example for each? (feel free to disagree and resolve this comment yourself)

I am providing the following as an example, please correct and adapt if it does not make sense:

`./benchmark dataset/small --mixed-workload 1000 20 70 10 0:` run on dataset `small` from folder `dataset`, total of 1000 queries 20% write etc..
`./benchmark dataset/small/read_specific --mixed-workload 1000 25 0 20 10 45`:  run on dataset `small` from folder `dataset`, total of 1000 queries, 45% will be from read_specific, 25% write, 20% update, 10% analytical

Reply:
I will add a bit more explanation in the next commit, and this  PR will have to follow up polish PR I will make this thing more straightforward.